client/src/components/CustomFields.js

```diff
@@ -167,9 +171,9 @@ const EnumField = fieldProps => {
 
 const enumHumanValue = (choices, fieldVal) => {
   if (Array.isArray(fieldVal)) {
-    return fieldVal && fieldVal.map(k => choices[k].label).join(", ")
+    return fieldVal && fieldVal.map(k => choices[k]?.label || "").join(", ")
```

Comment:
This will add just a comma and a space if the choice doesn't exist or has no label (e.g. `First Value, , Third Value`) which to me looks not so nice.